pkgs/applications/video/imagination/default.nix

```diff
@@ -0,0 +1,47 @@
+{ lib, stdenv, fetchurl, autoreconfHook, docbook_xsl, ffmpeg-full, glib, gtk3
+, intltool, libtool, libxslt, pkg-config, sox, wrapGAppsHook }:
+
+stdenv.mkDerivation rec {
+  pname = "imagination";
+  version = "3.6";
+
+  src = fetchurl {
+    url = "mirror://sourceforge/${pname}/${pname}-${version}.tar.gz";
+    sha256 = "139dgb9vfr2q7bxvjskykdz526xxwrn0bh463ir8m2p7rx5a3pw5";
+  };
+
+  nativeBuildInputs = [
+    autoreconfHook
+    docbook_xsl
+    intltool
+    libtool
+    libxslt
+    pkg-config
+    wrapGAppsHook
+  ];
+
+  buildInputs = [ ffmpeg-full glib gtk3 sox ];
+
+  preBuild = ''
+    substituteInPlace src/main-window.c --replace \
+      'gtk_icon_theme_load_icon(icon_theme,"image", 20, 0, NULL)' \
+      'gtk_icon_theme_load_icon(icon_theme,"insert-image", 20, 0, NULL)'
+    substituteInPlace src/main-window.c --replace \
+      'gtk_icon_theme_load_icon(icon_theme,"sound", 20, 0, NULL)' \
+      'gtk_icon_theme_load_icon(icon_theme,"audio-x-generic", 20, 0, NULL)'
+  '';
+
+  preFixup = ''
+    gappsWrapperArgs+=(
+       --prefix PATH : "${lib.makeBinPath [ ffmpeg-full sox ]}"
+    )
+  '';
+
+  meta = with lib; {
+    description = "Lightweight and simple DVD slide show maker";
+    homepage = "http://imagination.sourceforge.net";
+    license = licenses.gpl3Only;
```

Comment:
I see gpl2Plus here https://sourceforge.net/p/imagination/code/HEAD/tree/trunk/src/audio.c#l7

Reply:
Huh, the `COPYING` file in the 3.6 download GNU v3. What's the best way to handle that? List both?

Reply:
You should ask upstream.

Reply:
Heh, well I emailed the maintainer and got a nice reply back where he said, "I think it's GPL3 but don't worry, do whatever you want with the code, it's always open source!"